components/collapse/style/index.less

```diff
@@ -70,7 +70,7 @@
 
         .@{collapse-prefix-cls}-arrow {
           right: @padding-md;
-          left: initial;
+          left: inherit;
```

Comment:
The demo works fine.

<img width="1409" alt="image" src="https://user-images.githubusercontent.com/507615/59931064-c09bd280-9476-11e9-9ee3-e5d95f43f7b6.png">